.github/workflows/ci.yml

```diff
@@ -189,11 +189,17 @@ jobs:
           echo $RESPONSE
           ACCOUNT_INDEX=$(echo $RESPONSE | jq '.[0]' || exit 1)
           echo "::set-output name=account_index::$(echo ${ACCOUNT_INDEX})"
-          # TODO: Drop regions when multi-region is done
-          regions=("ap-northeast-1" "ap-northeast-2" "ap-northeast-3" "ap-south-1" "ap-southeast-1" "ap-southeast-2" "ca-central-1" "eu-central-1" "eu-north-1" "eu-west-1" "eu-west-2" "eu-west-3" "sa-east-1" "us-east-2" "us-west-1" "us-west-2")
-          regionslen=${#regions[@]}
-          index=$(($RANDOM % $regionslen))
-          region=${regions[$index]}
+
+          # temporary: fix codepipeline to us-east-1 , due to s3 limitations
```

Comment:
will this go away once s3 is multi-region?

Reply:
no, will go away once i implement https://github.com/iasql/iasql-engine/issues/1398